### PR TITLE
Create 3simransharma

### PR DESCRIPTION
**Pull request by 3simransharma**

### CODE REVIEW

File: data/3simransharma [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/3simransharma",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Check out my Design Portfolio on Behance",
+      "url": "https://www.behance.net/simransharma3",
+      "icon": "FaBehance"
+    },
```
Comment (khushalarora) [inline data/3simransharma:28]: Please remove the extra **' Comma '** from this line :+1: 
```Suggestion
    }
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hi @3simransharma, thank you for contributing :tada: 
Please rename your file and Add **`.json`** at the end :+1: 
It will look like **`data/3simransharma.json`**

**Comment by 3simransharma** [comment]
> Hi @3simransharma, thank you for contributing 🎉 Please rename your file and Add **`.json`** at the end 👍 It will look like **`data/3simransharma.json`**
Thank you for the review, I have changed the things you said.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**